src/main/java/com/spotify/reaper/storage/CassandraStorage.java

```diff
@@ -142,7 +153,7 @@ public boolean isStorageConnected() {
   @Override
   public boolean addCluster(Cluster cluster) {
     try {
-      session.execute(insertClusterPrepStmt.bind(cluster.getName(), cluster.getPartitioner(), cluster.getSeedHosts()));
+      session.executeAsync(insertClusterPrepStmt.bind(cluster.getName(), cluster.getPartitioner(), cluster.getSeedHosts()));
```

Comment:
> This will lead to potential silent failures and chained calls being inconsistent.

There is a risk of the async request silently failing. But as an idempotent request we expect it to retry.

The failure mode is interesting though. Already before this PR, there's no guarantee that the request completes (the jvm could crash, and/or it fails and an exception is thrown (and maybe it worked??).